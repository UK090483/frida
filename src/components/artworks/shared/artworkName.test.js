import React from "react"
import renderer from "react-test-renderer"
import ArtworkName from "./artworkName"
import "jest-styled-components"

describe("ArtworkName ", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ArtworkName />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
