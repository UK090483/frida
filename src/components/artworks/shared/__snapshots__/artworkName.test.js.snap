// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtworkName  renders correctly 1`] = `
.c0 {
  padding-top: 15px;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  min-height: 10px;
  margin-left: 29px;
}

.c1 {
  position: relative;
}

.c1::after {
  content: "";
  position: absolute;
  top: 50%;
  -webkit-transform: translateY(-50%) translateX(-30px);
  -ms-transform: translateY(-50%) translateX(-30px);
  transform: translateY(-50%) translateX(-30px);
  border-radius: 50%;
  left: 0;
  width: 20px;
  height: 20px;
  background-color: green;
}

<div
  className="c0"
>
  <span
    className="c1"
  >
    prop 
  </span>
  artworkName is missing
</div>
`;
